.github/workflows/ci.yml

```diff
@@ -9,9 +9,6 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@master
-      - name: Install ubuntu dependencies needed for Playwright
-        run: sudo apt-get install ffmpeg
```

Comment:
No longer need to manually install this after https://github.com/microsoft/playwright/pull/3882.